### DISCUSSION TITLE
Update fields_attributes

### DISCUSSION THREAD

**Discussion by alexduly** [mastodon/mastodon#24219] [Q&A]
Hi, 

Trying to update the fields_attributes using python request.fetch

Am a bit confused with the documentation, as it lists the fields_attributes under form_data parameters. 

However, I only seem to be able to get it working by including it as a string after the request string as the query params ( can't seem to get the payload option working either...). 

Example: 
`
 headers = {'Authorization': 'Bearer ' + mytoken}
data = {
'note' : 'my bio',
'display_name': 'My Name',
}
request.path(https://mastodon.example/api/v1/accounts/update_credentials
?fields_attributes[0][name]=Website
&fields_attributes[0][value]=https://mywebsite.com, headers=headers, data=data)`

TLDR: Can someone help me with the correct way to update the fields_attributes using the form data/ a dictionary if possible!

**Comment by alexduly** [comment]
Use json=data not data=data and it works...
